src/qt/bitcoingui.cpp

```diff
@@ -579,7 +579,7 @@ void BitcoinGUI::setClientModel(ClientModel *_clientModel)
         // Show progress dialog
         connect(_clientModel, &ClientModel::showProgress, this, &BitcoinGUI::showProgress);
 
-        rpcConsole->setClientModel(_clientModel);
+        rpcConsole->setClientModel(_clientModel, tip_info->block_height, tip_info->block_time, tip_info->verification_progress);
```

Comment:
Why not pass tip_info too?

Reply:
I think it's fine to pass primitives at this level to be more flexible with stricture changes.